remill/Arch/X86/Arch.cpp

```diff
@@ -477,35 +477,46 @@ static void DecodeImmediate(Instruction &inst,
                             xed_operand_enum_t op_name) {
   auto val = 0ULL;
   auto is_signed = false;
-  auto imm_size = xed_decoded_inst_get_immediate_width_bits(xedd);
   auto operand_size = xed_decoded_inst_get_operand_width(xedd);
 
-  CHECK(imm_size <= operand_size)
+  Operand op = {};
+  op.type = Operand::kTypeImmediate;
+  op.action = Operand::kActionRead;
+
+  if (XED_OPERAND_PTR == op_name) {
+    auto ptr_size = xed_decoded_inst_get_branch_displacement_width_bits(xedd);
+    CHECK(ptr_size <= operand_size)
+      << "Pointer size is greater than effective operand size at "
```

Comment:
Four space indent the `<<`, so it's like:
```c++
CHECK(...)
    << "hiiii";
```

Reply:
Done.